message_parser/src/test/java/cumulus_sled/message_parser/MessageParserTest.java

```diff
@@ -205,6 +205,31 @@ public void testException()
         }
     }
 
+
+    @Test
+    public void testLoggerWithCmaConfiguration() throws MessageAdapterException
+    {
+        final LoggerContext ctx = (LoggerContext) LogManager.getContext(false);
+        final Configuration config = ctx.getConfiguration();
+        TestAppender appender = (TestAppender) config.getAppenders().get("TestAppender");
+
+        MessageParser parser = new MessageParser(new TestEventMessageAdapter());
+        try
+        {
+            String inputJsonString = AdapterUtilities.loadResourceToString("parameter.input.json");
```

Comment:
I'm not seeing the `parameter.input.json` in this PR or elsewhere in the code base?

Reply:
That would help to have it included, yes 👍

Reply:
550cde1